client/.eslintrc.json

```diff
@@ -4,10 +4,14 @@
         "es2021": true,
         "node": true
     },
+    "globals": {
+        "localStorage": true
+    },
```

Comment:
Didn't know you could do this! Good catch, could you also refactor the other places that disable this error on a line by line basis?